ldk/javascript/src/types/oliveHelps/index.d.ts

```diff
@@ -423,6 +424,11 @@ declare namespace OliveHelps {
     tooltip?: string;
   };
 
+  type SectionTitle = Component<'sectionTitle'> & {
+    body: string;
+    textalign?: TextAlign;
```

Comment:
I thought  there is also a background style configuration in the card. Perhaps I'm misreading this:
`
Background strip color: Let's limit it to grey (#F5F5F5) and white 
`